### PR TITLE
SQL Question

### PR DESCRIPTION
**Pull request by Atharva-Morankar-09**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const sqlQuiz = [
+  {
+    Question:
+      "The maximum length of the varchar columns in MySQL is?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:4]:
```suggestion
      "In MySQL, what is the maximum string length for the VARCHAR data type?",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    Answer: "Upto 65,535 bytes",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:5]:
```suggestion
    Answer: "Up to 65,535 bytes",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+    Distractor1: "255 bytes",
+    Distractor2: "256 bytes",
+    Distractor3: "Upto 65,567 bytes",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:8]:
```suggestion
    Distractor3: "Up to 65,567 bytes",
```

File: src/data/sql-quiz.ts [hunk 4]
```diff
@@ -1,4 +1,15 @@
...
+    Explanation:
+      "Variable length string similar to CHAR(), has a maximum string length range from 0 to 65535.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:10]:
```suggestion
      "The VARCHAR  data type has a maximum string length range from 0 to 65535.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Atharva-Morankar-09** [comment]
Completed the requested changes. Thanks

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**